pkg/process/runner/submitter.go

```diff
@@ -456,6 +460,7 @@ func (s *CheckSubmitter) messagesToCheckResult(start time.Time, name string, mes
 		extraHeaders.Set(headers.ProcessVersionHeader, agentVersion.GetNumber())
 		extraHeaders.Set(headers.ContainerCountHeader, strconv.Itoa(getContainerCount(m)))
 		extraHeaders.Set(headers.ContentTypeHeader, headers.ProtobufContentType)
+		extraHeaders.Set(headers.AgentStartTime, strconv.Itoa(int(s.agentStartTime)))
```

Comment:
Optional nit: we're losing precision here, and as `Itoa()` is equivalent to `FormatInt(int64(i), 10)`, we're also doing unnecessary casts from int64 → int → int64. Let's just use `FormatInt()` directly